script.js

```diff
@@ -0,0 +1,170 @@
+ const jsonDados = [{
+     "title": "Work",
+     "timeframes": {
+         "daily": {
+             "current": 5,
+             "previous": 7
+         },
+         "weekly": {
+             "current": 32,
+             "previous": 36
+         },
+         "monthly": {
+             "current": 103,
+             "previous": 128
+         }
+     }
+ }, {
+     "title": "Play",
+     "timeframes": {
+         "daily": {
+             "current": 1,
+             "previous": 2
+         },
+         "weekly": {
+             "current": 10,
+             "previous": 8
+         },
+         "monthly": {
+             "current": 23,
+             "previous": 29
+         }
+     }
+ }, {
+     "title": "Study",
+     "timeframes": {
+         "daily": {
+             "current": 0,
+             "previous": 1
+         },
+         "weekly": {
+             "current": 4,
+             "previous": 7
+         },
+         "monthly": {
+             "current": 13,
+             "previous": 19
+         }
+     }
+ }, {
+     "title": "Exercise",
+     "timeframes": {
+         "daily": {
+             "current": 1,
+             "previous": 1
+         },
+         "weekly": {
+             "current": 4,
+             "previous": 5
+         },
+         "monthly": {
+             "current": 11,
+             "previous": 18
+         }
+     }
+ }, {
+     "title": "Social",
+     "timeframes": {
+         "daily": {
+             "current": 1,
+             "previous": 3
+         },
+         "weekly": {
+             "current": 5,
+             "previous": 10
+         },
+         "monthly": {
+             "current": 21,
+             "previous": 23
+         }
+     }
+ }, {
+     "title": "Self Care",
+     "timeframes": {
+         "daily": {
+             "current": 0,
+             "previous": 1
+         },
+         "weekly": {
+             "current": 2,
+             "previous": 2
+         },
+         "monthly": {
+             "current": 7,
+             "previous": 11
+         }
+     }
+ }]
+ const data = JSON.parse(JSON.stringify(jsonDados))
+ console.log(data)
+ let work = data[0].timeframes
+ console.log(work)
+ let play = data[1].timeframes
+ let study = data[2].timeframes
+ let exercise = data[3].timeframes
+ let social = data[4].timeframes
+ let selfCare = data[5].timeframes
```

Comment:
```suggestion
const dados = {
   work: data[0].timeframes,
   play = data[1].timeframes
   study = data[2].timeframes
   exercise = data[3].timeframes
   social = data[4].timeframes
   selfCare = data[5].timeframes
}
 
```
Ao invés de criar as 6 variáveis diferentes, fica mais fácil criar um único objeto agregando todas.